### ISSUE TITLE
Citations/References: some references don't have a backlink

### ISSUE THREAD

**Issue by sk1p** [LiberTEM/LiberTEM#512]
Some citations don't have a backlink, while others do. This needs to be debugged and possibly fixed (maybe upstream in the bibtex support of sphinx)

**Comment by sk1p** [comment]
@uellue do you have an idea about this one? See https://libertem.github.io/LiberTEM/index.html#citations - First item, Gibson1997 is cited from the FEM application docs, but no link back. Is it possible that `.. bibliography::` only supports anchors in the same document?

**Comment by uellue** [comment]
@sk1p Yes, back links seem to work only in the same document. The forward links work, though. Currently I don't have a good idea how to fix this. Adding a bibliography per document doesn't work.

I've raised the issue upstream: https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/194

**Comment by uellue** [comment]
Wontfix for now, larger upstream issue: https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/194#issuecomment-743761553

**Comment by uellue** [comment]
Closing, Not much we can do I guess.
